### DISCUSSION TITLE
Is there a way to communicate with Supervisor API?

### DISCUSSION THREAD

**Discussion by poldim** [zachowj/node-red-contrib-home-assistant-websocket#655] [Q&A]
In the [HA Docs](https://developers.home-assistant.io/docs/api/supervisor/examples) it shows how you can communicate with your instance.  I have tested this by running `curl -sSL http://supervisor/supervisor/ping` from terminal on the RPi.  I'd like to initiate a command to the supervisor from a NR flow.

Is there any way to do this?

**Comment by zachowj** [comment]
I don't run supervisor but should be able to use a request node.
